src/fragments/lib/push-notifications/ios/setup_push_service/setup-apns.mdx

```diff
@@ -26,41 +26,39 @@ Before you begin, you must have an account with the Apple Developer Program as a
 
 Create an app ID to identify your app in the Apple Developer portal. You need this ID when you create an SSL certificate for sending push notifications, when you create an iOS distribution certificate, and when you create a provisioning profile.
 
-If you already have an ID assigned to your app, you can skip this step. You can use an existing app ID provided it doesn't contain a wildcard character ("\*").
+_If you already have an ID assigned to your app, you can skip this step. You can use an existing app ID provided it doesn't contain a wildcard character ("\*")._
```

Comment:
Why the italics? It doesn't read better IMO. 
If we want to make it more prominent we can use a `<Callout>` instead.

This comment applies to the other sections were italics was used.

Reply:
I'll remove the italics 👍